.circleci/config.yml

```diff
@@ -1091,13 +1091,13 @@ workflows:
       #          requires:
       #            - wait-calypso-live
       #          test-flags: '-C -S $CIRCLE_SHA1'
-      - test-e2e-canary:
-          name: test-e2e-canary-ie
-          requires:
-            - wait-calypso-live
-          test-flags: '-z -S $CIRCLE_SHA1'
-          test-target: 'IE11'
-          slack-webhook: '$SLACK_IE'
+      # - test-e2e-canary:
```

Comment:
I think we should delete this, If anyone wants to bring it back, it is safely stored in git history anyway.

Reply:
Got it.